docs/reference/package-control/channel.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Package Control: channel.json"
+---
+
+<!-- Original: https://github.com/wbond/package_control/blob/master/example-channel.json -->
+
+
+# Examples for the channel.json
+
+The latest version of the default `channel.json`
```

Comment:
The default seed channel is https://github.com/wbond/package_control_channel/blob/master/channel.json.  It only lists registered repositories.  A crawler compiles this to a channel suitable for consumption by the Package Control client running in Sublime Text.  I.e. it visits all listed repositories and checks all packages in those repositories for the latest releases and the current metadata.  

The latest version of our compiled `channel.json` ...

Reply:
I'm learning a lot by doing these docs! Pushed an update that incorporates your explanation and hopefully describes it better.